src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -322,8 +351,22 @@ export default class FilterOptionsComponent extends Component {
   }
 
   apply () {
-    const filter = this._buildFilter();
-    this.core.setFilter(this.name, filter);
+    switch (this.config.optionType) {
+      case OptionTypes.RADIUS_FILTER:
```

Comment:
Minor nit, but could consolidate things a bit here. 
const selectedOption = this.config.options.find(o => o.selected && o.value !== 0);
selectedOption && this.core.setLocationRadius(selectedOption.value);

Reply:
I feel like this is not logically equivalent. If the selected value is === 0 I still want to call core.clearLocationRadius. I could move the "if the value is 0 clear the locationRadius" logic into core.setLocationRadius and delete core.clearLocationRadius though, if you like that better? The backend will throw an error if you set locationRadius to 0 so I think that makes sense, but maybe we don't want to tie our behavior with theirs.